src/closures/constant_smagorinsky.jl

```diff
@@ -0,0 +1,89 @@
+Base.@kwdef struct ConstantSmagorinsky{T} <: ScalarDiffusivity{T}
+               C :: T = 0.23
+         prandtl :: T = 1.0
```

Comment:
Would `Pr` be a better variable name?

Reply:
`Pr` is fine with me. The rest of the code tends to use whole words (`boundary_conditions` rather than `bcs`...) so I tried to be consistent with what I perceived to be `Oceananigans` style.

Reply:
I usually just go with what looks more readable in the code so any convention was probably accidental but probably not a bad idea.